packages/phishing-controller/src/PhishingController.ts

```diff
@@ -392,10 +454,6 @@ export class PhishingController extends BaseController<
    * this function that prevents redundant configuration updates.
    */
   async #updateStalelist() {
-    if (this.disabled) {
```

Comment:
Interestingly, we never configure this controller as "disabled" on extension or mobile. Removing this seems appropriate, it looks like an unused feature.